cfe_internal/enterprise/federation/federation.cf

```diff
@@ -673,9 +707,19 @@ bundle agent entry
     enabled.am_on.am_superhub.!am_paused::
       "CFEngine Enterprise Federation Feeder Data Import"
         handle => "imported_data",
-        depends_on => { "transport_user", "ensure_feeders" },
+        depends_on => { "transport_user", "ensure_feeders", "data_transport" },
```

Comment:
this makes importing data depend on having data to transport right? What are the chances that there is data to import from previous transfer and no new data to transfer?
So, data transferred, then i dunoo, someone pkills the wrong process mid import, then agent run again.

Reply:
Yes, but it also makes importing REQUIRE data transfer first.

I don't know that this will for sure cause any issue. But it is adding a strong dependancy between transfer and import that may not be desired. E.g. what if I am sneaker netting data from some hub and dropping it into place on the superhub for import. (this is a completely imagined issue of course and no custoemr yet has said they want to do this.

So, I am not completely against the dependancy, just want to make sure that we at least consider if there are any cases where we want to import even if data transpoert has not happened or is unsuccessful in some way.